examples/mutators/set-namespace/README.md

```diff
@@ -1,5 +1,8 @@
 # Set Namespace
```

Comment:
@mengqiy @etefera I suggest authoring a doc using one of the functions as an example that includes all the content we have talked about:
- Function reference generated from --help
- One or more examples
- Category of function
- Link to source code (if open source)
- etc.

Once we have the desired UX hashed out for one function, then things fall into place. As it stands, the example README is being shown as the entirety of documentation for a function which is confusing.

Reply:
Updated site according to doc.